lib/bulkPrefixer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Ahah! the elasticsearch bulk API allows you to load many documents with one request, to do so you need to prefix the document with bulk loading information (in this case, just that we want to index the document)